src/qt/bitcoingui.cpp

```diff
@@ -769,6 +768,7 @@ void BitcoinGUI::setCurrentWallet(WalletModel* wallet_model)
             break;
         }
     }
+    setWalletActionsEnabled(true);
```

Comment:
This shouldn't be necessary.

Reply:
I'll check, thanks.

Reply:
Sorry, that was because `setWalletActionsEnabled` was not working properly, and `setWalletActionsEnabled` was not working because I misread your suggestion and I was still calling the function `enableHistoryAction` incorrectly instead of just using directly `historyAction->setEnabled`. It fixed now in 260d6eb. Thanks!